config/routes.rb

```diff
@@ -175,6 +181,7 @@
       resources :recruitment_cycles,
                 only: :show,
                 param: :year do
+        resources :courses, controller: "course_searches"
```

Comment:
For providers, the search was incorporated as part of the [existing `CourseController`](https://github.com/DFE-Digital/teacher-training-api/pull/1144/files#diff-63584eeba1d49b4829082079274d0447R9), I like this because we should support searches, etc, on any `Course` index.

Can we just add a `courses` resource here that uses the existing `CourseController` and add search support to it?

Reply:
I've updated this after offline discussion. This is now `/api/V3/courses` and only routes `index`.